types/aws-lambda/package.json

```diff
@@ -1,7 +1,7 @@
 {
     "private": true,
     "name": "@types/aws-lambda",
-    "version": "8.10.9999",
+    "version": "8.11.9999",
```

Comment:
TLDR: leave this unchanged. 

It's currently the last lambda runtime version that made a potentially breaking change (async handlers), back when that was pretty often.

I do want to fix this by breaking up this package so we get meaningful versions again, but it's a bit scary and I'm very lazy 😅

Reply:
Tracking - I put it back the way it was. I totally get it :)